### DISCUSSION TITLE
A multi-lingual Github

### DISCUSSION THREAD

**Discussion by zc5162** [community/community#101646] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Github needs more languages to attract developers and users from all over the world。
![image](https://github.com/community/community/assets/139825029/1f85d451-e8cd-4683-89bb-0b68f28f725f)

**Comment by zc5162** [comment]
Image is from https://www.un.org/

**Comment by spenserblack** [comment]
Related: https://github.blog/2010-07-13-github-in-your-language/

**Reply by zc5162** [reply]
If GitHub have Chinese, It will be good.(I'm a Chinese)
